backend/apps/mentorship/models/mentee_module.py

```diff
@@ -0,0 +1,39 @@
+"""Enrollment model for the Mentorship app."""
+
+from __future__ import annotations
+
+from django.db import models
+
+from apps.common.models import TimestampedModel
+from apps.mentorship.models.common import ExperienceLevel, MatchingAttributes
+
+
+class MenteeModule(TimestampedModel, ExperienceLevel, MatchingAttributes):
```

Comment:
Could we remove `ExperienceLevel, MatchingAttributes` from here? 
```suggestion
class MenteeModule(TimestampedModel):
```